bot/cogs/reminders.py

```diff
@@ -101,10 +101,13 @@ def __init__(self, bot: Bot):
         embed.colour = Colour.blurple()
         embed.set_author(
             icon_url=Icons.remind_blurple,
-            name="It has arrived!"
-        )
+            name="It has arrived!")
```

Comment:
Why this change?

Reply:
I reverted a change here and I probably forgot to re-add the line break.